extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -160,17 +170,19 @@ public function editBoardModal() {
 		if ( !$this->wg->User->isAllowed( 'forumadmin' ) ) {
 			$this->displayRestrictionError();
 			wfProfileOut( __METHOD__ );
+
 			return false;
 			// skip rendering
 		}
 
-		$boardId = $this->getVal( 'boardId', -1 );
+		$boardId = $this->getVal( 'boardId', - 1 );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "boardId" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)